src/public/less/bootstrap/close.less

```diff
@@ -29,6 +29,6 @@
     cursor: pointer;
     background: transparent;
     border: 0;
-    -webkit-appearance: none;
```

Comment:
The files in `bootstrap` are just a copy of twitter's bootstrap project and we shouldn't modify those.